### PR TITLE
FIX: more fixes and cleanup

### PR DESCRIPTION
**Pull request by klauer**
Continuing the messy changes of #80 and #81:

~* Attempted to address #82 but failed~
* Put all `PYTEST_ARGS` in one line
* Fix normal Python builds as `pytest-cov` was not being installed

### CODE REVIEW

**Comment by klauer** [comment]
@ZLLentz any idea how to constrain the conda build to a specific Python version if the recipe only indicates a lower bound? (e.g., `meta.yaml: python >= 3.7` and we specifically want Python 3.9 for our "latest" test env)

**Comment by klauer** [comment]
I tried `--python` in [5686363](https://github.com/pcdshub/pcds-ci-helpers/pull/83/commits/56863630514a5cd238570e4675176c6a39ab08c6) but found that it ignored it and still built with Python 3.10. Perhaps I did something else wrong?

**Comment by ZLLentz** [comment]
That's annoying, I would have expected that to work.

**Comment by klauer** [comment]
I never resolved the 3.9-only-build issue, but this has some fixes that are relevant for other Python packages. If all looks good, we should merge this.

File: travis/shared_configs/anaconda-build.yml [hunk 1]
```diff
@@ -28,6 +30,7 @@ jobs:
           - micromamba info
           - micromamba config list
           # boa is the mamba equivalent of conda-build and provides the conda mambabuild command
+          # Constrain ourselves to ${PYTHON_VERSION} in case this package is version-dependent
```
Comment (klauer) [inline travis/shared_configs/anaconda-build.yml:33]:
```suggestion
```

File: travis/shared_configs/anaconda-build.yml [hunk 2]
```diff
@@ -4,6 +4,8 @@ jobs:
   include:
       - stage: build
         name: "Anaconda Build"
+        env:
+          - PYTHON_VERSION: 3.9
```
Comment (klauer) [inline travis/shared_configs/anaconda-build.yml:8]:
```suggestion
```

File: travis/shared_configs/python-tester-conda.yml [hunk 3]
```diff
@@ -33,17 +33,18 @@ jobs:
         - echo "Conda Requirements':' ${CONDA_REQUIREMENTS:=dev-requirements.txt}"
         - echo "Conda packages installed for CI':' ${CONDA_CI_PACKAGES:=pytest-cov}"
         # Manage conda environment
-        - micromamba create -n ${CONDA_ENV_NAME} python=$PYTHON_VERSION ${CONDA_PACKAGE} ${CONDA_EXTRAS} --file ${CONDA_REQUIREMENTS}
+        - micromamba create -n ${CONDA_ENV_NAME} python=$PYTHON_VERSION ${CONDA_PACKAGE} ${CONDA_EXTRAS} ${CONDA_CI_PACKAGES} --file ${CONDA_REQUIREMENTS}
```
Comment (klauer) [inline travis/shared_configs/python-tester-conda.yml:36]: `pytest-cov` was missing for `python-tester-conda`-based builds prior to this change

**Review by ZLLentz — APPROVED** [review]
Let's merge this for all the other fixes :+1:

**Pull request merged**